charts/kafka-operator/templates/operator-deployment-with-webhook.yaml

```diff
@@ -37,7 +37,7 @@ webhooks:
     service:
       name: "{{ include "kafka-operator.fullname" . }}-operator"
       namespace: {{ .Release.Namespace }}
-      path: /validate
+      path: /validate-kafka.banzaicloud-io-v1alpha1-kafkatopic
```

Comment:
This change in webhook path needs to be reflected in https://github.com/banzaicloud/koperator/tree/master/config/base/webhook as well

Reply:
According to this https://github.com/banzaicloud/koperator/pull/832#discussion_r920871898
I think problem can only happen if the operator is using another version of the KafkaCluster or KafkaTopic then what is specified in the validation admission webhook resource.